expression/builtin_arithmetic.go

```diff
@@ -497,13 +497,11 @@ func (c *arithmeticDivideFunctionClass) getFunction(args []Expression, ctx conte
 		bf := newBaseBuiltinFuncWithTp(args, ctx, tpReal, tpReal, tpReal)
 		c.setType4DivReal(bf.tp)
 		sig := &builtinArithmeticDivideRealSig{baseRealBuiltinFunc{bf}}
-		sig.setPbCode(tipb.ScalarFuncSig_MultiplyReal)
```

Comment:
forget to setPbCode(ipb.ScalarFuncSig_DivReal) ?

Reply:
There is not this sig in pb